rif.config.js

```diff
@@ -10,18 +10,18 @@ module.exports = {
   },
   rns: {
     contracts: {
-      rns: '',
-      registrar: '',
-      reverseRegistrar: '',
-      publicResolver: '',
-      nameResolver: '',
-      multiChainResolver: '',
-      rif: '',
-      fifsRegistrar: '',
-      fifsAddrRegistrar: '',
-      rskOwner: '',
-      renewer: '',
-      stringResolver: '',
+      rns: '0xDA7Ce79725418F4F6E13Bf5F520C89Cec5f6A974',
```

Comment:
This file not be commited too

Reply:
I told him we can remove it later but if you want we can do it here, anyway this file will be overwritten by gulp so i think it doesn't matter, anyway to keep the line of work i think you can remove the changes here @fmelo, you can do that by reseting this file with git. We should have this file in the .gitignore file anyway

Reply:
Same here guys, fixed